qoqo/tests/integration/measurements/mod.rs

```diff
@@ -10,6 +10,8 @@
 // express or implied. See the License for the specific language governing permissions and
 // limitations under the License.
 
+#![allow(clippy::redundant_clone)]
```

Comment:
Why has this been added?

Reply:
it suppresses clippy warnings in the test regarding duplicate clones, they are generated when `.clone()` is tested. 

It's just a preference of mine but I can revert it, it's not necessary of course.

Reply:
Fixed with new solution.